arch/M68K/M68KDisassembler.h

```diff
@@ -26,5 +26,6 @@ typedef struct m68k_info {
 } m68k_info;
 
 bool M68K_getInstruction(csh ud, const uint8_t* code, size_t code_len, MCInst* instr, uint16_t* size, uint64_t address, void* info);
+void get_ea_mode_op(m68k_info *info, cs_m68k_op* op, unsigned int instruction, unsigned int size);
```

Comment:
actually this function is only used in M68KDisassembler.c, so we better turn it to static?